app/styleguide/components/s-palette-list.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div :class="b('')">
+    <s-palette-item
+      v-for="palette in palettes"
+      :palette="palette"
+      :colors="colors[palette]"
+      :key="palette"/>
+  </div>
+</template>
+
+<script>
+
+  import _colors from '../../setup/js/color';
```

Comment:
- [ ] Please don't use `underline` for variable names. In JS this often is seen as a private variable, which we don't use.